desktop/api/src/main/java/org/datacleaner/windows/DataSetWindow.java

```diff
@@ -174,7 +174,7 @@ protected void done() {
                     logger.error("Unexpected error occurred while building DataSetWindow contents", e);
                     DataSetWindow.this.dispose();
                     WidgetUtils.showErrorMessage("Unexpected error",
-                            "An unexpected error occurred while building data set. See logs for details.");
+                            "An unexpected error occurred while building data set. See logs for details." + e.getMessage());
```

Comment:
This error message is deliberately not showing the exception message because it can be _anything_. So reintroducing it is not good - it should be hidden from the user since most users don't want a very techical message thrown in their head anyway.

Besides that - it's also missing a space or else it would append it directly onto the dot ;-)